tutorials/examples/train_ising.py

```diff
@@ -26,9 +26,6 @@ def main(args):
 
     n_hidden = 2
     acc_fn = "relu"
-    lr = 0.001
-    lr_Z = 0.01
-    validation_samples = 1000
```

Comment:
Don't do this please. Variables should be named somehow to make it easier for the reader. I don't like magic numbers hanging around in scripts -- it requires more work to understand what they are for.

Reply:
These variables are not used

Only `lr` could be inserted at line 73, but it has a different value, and I believe it is clearer as it is:
```python
optimizer = torch.optim.Adam(gflownet.parameters(), lr=1e-3)
```

Reply:
Actually, https://github.com/GFNOrg/torchgfn/pull/203 inserted lr in that line
(what are the odds)

Reply:
OK - I must have lost track - thanks for following up on this.